src/app/about/page.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+import styles from "./About.module.css";
+
+export default function About() {
+  return (
+    <main className={styles.about}>
+      <h1 className={styles.title}>About</h1>
+      <div className={styles.descriptionContainer}>
+        <p className={styles.description}>
+          We are an organization that helps students learn and apply software
+          engineering principles to real-world applications. We host weekly
+          workshops on topics like software design to help bridge the gap
+          between what students need to know for industry and what they&apos;re
+          taught in classes. These concepts help with building complex software
+          systems and better prepare members for team projects, internships, and
+          careers in software development.
+        </p>
+      </div>
+
+      <div className={styles.imageContainer}>
+        <Image
+          src="group_photo.jpg"
+          alt="fall_2022_planning_meeting.jpg
```

Comment:
[Please Change]: The file name should not be part of the `alt` text; I was referring to updating the name of the file itself. Apologies if my example was unclear.